Leaflet.Graticule.js

```diff
@@ -490,8 +551,8 @@ L.LatLngGraticule = L.Layer.extend({
                     ctx.stroke();
 
                     if (self.options.showLabel && label) {
-                        ctx.fillText(lngstr, tt.x - (txtWidth/2), txtHeight+1);
-                        ctx.fillText(lngstr, bb.x - (txtWidth/2), hh-3);
+                        ctx.fillText(lngstr, tt.x + 3, txtHeight+1);
```

Comment:
Why this change? Seems unrelated at first sight.

Reply:
This change is about labels placements.